netci.groovy

```diff
@@ -28,6 +28,9 @@ def static getBuildJobName(def configuration, def os) {
             if (os == 'Windows_NT') {
                 buildCommand = ".\\build.cmd -Configuration ${lowerConfiguration} Default"
             }
+            else if (os == 'Windows_2016') {
+                buildCommand = ".\\build.cmd -Configuration ${lowerConfiguration} -RunInstallerTestsInDocker Default"
+            }
```

Comment:
This is going to make some CI to run on `Windows_NT` and some on `Windows_2016`. And the MSI tests will not be deterministic-ally run every time. 

We can do all the CI runs only on `Windows_2016`.